### DISCUSSION TITLE
Batch Notification Settings Modification

### DISCUSSION THREAD

**Discussion by wltjr** [community/community#69705] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I watched too many repos. I now need to unwatch a bunch, hundreds, and I can only unwatch them all, or do it one by one :weary: . It would really be great if there was a check box so I could select batches of repos at a time to unwatch and be selective while doing batch operation to unwatch many, instead of all at once or one at a time, need something in between. Thank you!

**Comment by ebubekirgungor** [comment]
Hello, I generated a html file to do this with ChatGPT 3.5 September 25 Version.
You should generate a new api token in https://github.com/settings/tokens
Then put your api token to this file in line 22.
Finally change the extension to html and open it. You will see a table of repositories that you are watching and checkboxes to unwatch them.
[watch.txt](https://github.com/community/community/files/12859489/watch.txt)
